server/package.json

```diff
@@ -61,10 +63,12 @@
     "got": "^11.8.2",
     "hardhat": "^2.6.0",
     "js-yaml": "^4.1.0",
+    "lodash": "^4.17.21",
```

Comment:
I am not opposed to lodash, but it does change what is idiomatic in the codebase. Put another way you can get really function in style (which I like). Lets confirm we are happy to take this turn with @alcuadrado